### DISCUSSION TITLE
The titleBarStyle window option in package.json for electron-app does not get applied on Linux.

### DISCUSSION THREAD

**Discussion by aghayanlena** [eclipse-theia/theia#13565] [General] [answered]
I have the following configuration in the package.json file of electron-app: 
```
"theia": {
    "target": "electron",
     "frontend": {
      "config": {
        "applicationName": "Custom electron-app",
        **"electron": {
            "windowOptions": {
                "titleBarStyle": "custom"
            }**
        }
      }
    }
```
    
  However, it does not overwrite the default native title bar style. It only gets updated when the frontend style is changed from the application. 
  
 Can we update the default value of the titleBarStyle during the first execution of the electron app?

**Comment by msujew** [comment, marked as answer]
Hey @aghayanlena,

You need to set the correct preference in the package.json for this to correctly take effect:

```json
{
  "theia": {
    "target": "browser",
    "frontend": {
      "config": {
        "applicationName": "Custom electron-app",
        "preferences": {
          "window.titleBarStyle": "custom"
        }
      }
    }
  }
}
```

**Reply by aghayanlena** [reply]
Hi @msujew , 

I build an electron-app, not the browser-app. 
But anyway, the following configuration also does not change the default title style too: 
```
{
  "theia": {
    "target": "electron",
    "frontend": {
      "config": {
        "applicationName": "Custom electron-app",
        "preferences": {
          "window.titleBarStyle": "custom"
        }
      }
    }
  }
}
```

**Reply by msujew** [reply]
Have you deleted the electron storage? The title bar style will be saved there if you have switched it manually before.

**Reply by aghayanlena** [reply]
No, I have not. Where the electron storage is located?

**Reply by msujew** [reply]
It's located in a `config.json` file in [this directory](https://www.electronjs.org/docs/latest/api/app#appgetpathname) (the actual directory depends on the app name).

**Reply by aghayanlena** [reply]
After removing the electron storage, the default value gets updated. Thanks @msujew !
